src/v2/Components/Mixins.tsx

```diff
@@ -1,48 +1,47 @@
 import colors from "v2/Assets/Colors"
 import { garamond } from "v2/Assets/Fonts"
 import { css } from "styled-components"
-import { InputProps } from "./Input"
-
-export const borderedInput = (props: InputProps & BorderProps = {}) => {
-  return css`
-    padding: 10px;
-    box-shadow: none;
-    transition: border-color 0.25s;
-    margin-right: 10px;
-    resize: none;
-    display: block;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-
-    &::placeholder {
-      color: ${colors.grayMedium};
-      text-overflow: ellipsis;
-      line-height: normal;
-    }
-
-    ${garamond("s17")};
-    ${border(props)};
-  `
-}
 
 export interface BorderProps {
   hasError?: boolean
 }
-export const border = (props: BorderProps = {}) => {
-  return css`
-    border: 1px solid ${props.hasError ? colors.redMedium : colors.grayRegular};
-    transition: border-color 0.25s;
 
-    &:hover,
-    &:focus,
-    &.focused {
-      border-color: ${props.hasError ? colors.redMedium : colors.purpleRegular};
-      outline: 0;
-    }
+export const borderMixin = css`
+  border: 1px solid
+    ${({ hasError }: BorderProps) =>
+      hasError ? colors.redMedium : colors.grayRegular};
+  transition: border-color 0.25s;
 
-    &:disabled {
-      border: 2px dotted ${colors.grayRegular};
-    }
-  `
-}
+  &:hover,
+  &:focus,
+  &.focused {
+    border-color: ${({ hasError }: BorderProps) =>
+      hasError ? colors.redMedium : colors.purpleRegular};
+    outline: 0;
+  }
+
+  &:disabled {
+    border: 2px dotted ${colors.grayRegular};
+  }
+`
+
+export const borderedInputMixin = css`
+  padding: 10px;
+  box-shadow: none;
+  transition: border-color 0.25s;
+  margin-right: 10px;
+  resize: none;
+  display: block;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+
+  &::placeholder {
+    color: ${colors.grayMedium};
+    text-overflow: ellipsis;
+    line-height: normal;
+  }
+
+  ${garamond("s17")};
+  ${borderMixin};
+`
```

Comment:
So these were the only things that are a little bit different. I just made them normal mixins instead of functions that return mixins. I *think* this was done so as to support required types, and there might be a way to do this properly, but since these are dead I just removed the wrapping function and typed the internal props functions manually.